python/bridgestan/model.py

```diff
@@ -183,8 +216,8 @@ def __init__(
             ctypes.c_void_p,
             ctypes.c_int,
             ctypes.c_int,
-            double_array,
-            double_array,
+            generic_double_array,
+            generic_double_array,
```

Comment:
Maybe I missed the point on the type `unc_p_double_array`.  It seems like this should have the type associated with the variable name `unc_p_double_array` even though this name doesn't fit this case.  But shouldn't this out variable named `out` strictly be `WRITABLE`?